Sources/ServiceLifecycle/ServiceRunnerConfiguration.swift

```diff
@@ -0,0 +1,49 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftServiceLifecycle open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftServiceLifecycle project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftServiceLifecycle project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import UnixSignals
+
+/// The configuration for the ``ServiceRunner``.
+public struct ServiceRunnerConfiguration: Hashable, Sendable {
+    /// The runner's logging configuration.
+    public struct LoggingConfiguration: Hashable, Sendable {
```

Comment:
Is this just for keys? If so we should probably change the name of this config object to include `Keys` and then we can drop the `Key` suffix from each property.

Reply:
I wanted to keep it open for more logging config if we ever want to add something.

Reply:
maybe put the keys under a Keys "container" then